apps/client/app/components/EmailPreviewer.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { useState } from "react";
 import { render } from "@react-email/render";
 import WelcomeEmail from "../components/emails/WelcomeEmail.jsx";
-import "../styles/previwer.css";
+import "./previwer.css";
```

Comment:
Minor thing concerning the name of this file, could you rename it ?
`import "./previewer.css";`

Reply:
thanks for noticing, will change it